lib/Ix/JMAP/SentenceBroker.pm

```diff
@@ -11,10 +12,29 @@ use JMAP::Tester::Response::Sentence;
 use JMAP::Tester::Response::Paragraph;
 use JSON::Typist;
 
+=head1 OVERVIEW
+
+This is an implementation of L<JMAP::Tester::Role::SentenceBroker>.
+Nothing much uses it, but you can if you like.
```

Comment:
Ix::Processor::JMAP uses this

Reply:
(through Ix::JMAP::SentenceCollection

Reply:
Oh yeah, I just missed the role implementation in SentenceCollection and nothing in the code here ever calls ->broker, so I overlooked it. Have tweaked.